game/world/worldobjects.cpp

```diff
@@ -995,6 +999,27 @@ static bool checkFlag(Interactive& i,WorldObjects::SearchFlg f){
   return true;
   }
 
+template<class T>
+bool checkTargetType(T&, TargetType) { return true; }
+
+static bool checkTargetType(Npc& n, TargetType t) {
+  if(bool(t&(TARGET_TYPE_ALL|TARGET_TYPE_NPCS)))
```

Comment:
Is this code also G1 only?

Reply:
No can be used for G2. `SPL_WindFist` has `TARGET_TYPE_NPCS`. If changed to `TARGET_TYPE_UNDEAD` novices can't be targeted anymore.

Here it's just a leftover because  non-npcs can't be targeted anyway. Mostly if G2 does something different G1 stuff is still functional but unused,